example/javascriptlib/testing/1-test-suite/build.mill

```diff
@@ -0,0 +1,22 @@
+package build
+
+import mill._, javascriptlib._, javascriptlib.testmods._
+
+object bar extends JestModule
```

Comment:
Can we have each test module nested-within and dependent-upon a normal `TypeScriptModule`? That would match the other examples for `scalalib`/`javalib`/`kotlinlib`